src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -2,37 +2,11 @@
 
 import java.util.List;
 
-/**
- * Parameterize this service and add its implementation in a separate class.
- */
-public interface MachineService {
+public interface MachineService<T> {
```

Comment:
```suggestion
public interface MachineService<T extends Machine> {
```
better to restrict usage of generics so you won't be able to user MachineService<Cat> etc